src/main/external-resources/renderers/DefaultRenderer.conf

```diff
@@ -621,8 +621,10 @@ AutoExifRotate =
 #    mkv      Matroska Video                                video/x-matroska
 #    mlp      Meridian Lossless Packing                     audio/vnd.dolby.mlp
 #    mov      QuickTime File Format                         video/quicktime
+#    mp2      MPEG Audio Layer II                           audio/mpeg
```

Comment:
@Sami32 I would be much easier if you created a "normal" comment (not a code-comment) and listed all the issues you found in one place, maybe even using "tasks":
- [ ] Fix issue 1
- [ ] Fix issue 2

That way I wouldn't have to look through all the comments 50 times to find them all.